Documentation/commands.md

```diff
@@ -317,6 +316,42 @@ Work in progress. Please contribute!
 
 Work in progress. Please contribute!
 
+### Logging
+
+By default, rkt will send logs directly to stdout/stderr, allowing them to be caputered by the invoking process.
+On host systems running systemd, rkt will attempt to integrate with journald on the host.
```

Comment:
Right.

I just want to avoid the predictable bug report that it will not work for people trying the next release of rkt (compiled with flavor=coreos, that is systemd-v215). I am not sure how to phrase it.